Source/WebCore/dom/ScriptExecutionContext.h

```diff
@@ -409,7 +409,7 @@ class ScriptExecutionContext : public SecurityContext, public TimerAlignment {
     RejectedPromiseTracker* ensureRejectedPromiseTrackerSlow();
 
     void checkConsistency() const;
-    RefCountedSerialFunctionDispatcher& nativePromiseDispatcher();
+    WEBCORE_EXPORT RefCountedSerialFunctionDispatcher& nativePromiseDispatcher();
```

Comment:
Is this needed?

Reply:
Yes, for using enqueueTaskWhenSettled within Internals.